swagger.yaml

```diff
@@ -3498,7 +3498,88 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /v2/clusters/{cluster_id}/ignored-validations:
+    get:
+      tags:
+        - installer
+      description: Fetch the validations which are to be ignored for this cluster.
+      operationId: v2GetIgnoredValidations
+      parameters:
+        - in: path
+          name: cluster_id
+          description: The cluster whose failing validations should be ignored according to this list.
+          type: string
+          format: uuid
+          required: true
+      responses:
+        "200":
+          description: Success.
+          schema:
+            $ref: '#/definitions/ignored-validations'
+        "400":
+          description: Error.
+          schema:
+            $ref: '#/definitions/error'
+        "404":
+          description: Error.
+          schema:
+            $ref: '#/definitions/error'
+    put:
+      tags:
+        - installer
+      description: Register the validations which are to be ignored for this cluster.
+      operationId: v2SetIgnoredValidations
+      parameters:
+        - in: path
+          name: cluster_id
+          description: The cluster whose failing validations should be ignored according to this list.
+          type: string
+          format: uuid
+          required: true
+        - in: body
+          name: ignored_validations
+          description: The validations to be ignored.
+          required: true
+          schema:
+            $ref: '#/definitions/ignored-validations'
+      responses:
+        "201":
+          description: Success.
+          schema:
+            $ref: '#/definitions/cluster' 
+        "400":
+          description: Error.
+          schema:
+            $ref: '#/definitions/error'
+        "404":
+          description: Error.
+          schema:
+            $ref: '#/definitions/error'
+        "401":
+          description: Unauthorized.
+          schema:
+            $ref: '#/definitions/infra_error'
+        "403":
+          description: Forbidden.
+          schema:
+            $ref: '#/definitions/infra_error'
+        "500":
+          description: Error.
+          schema:
+            $ref: '#/definitions/error'
+
 definitions:
+  ignored-validations:
```

Comment:
shouldn't the `cluster:` definition have `ignored-validations` under it?

Reply:
We have a `cluster:` [object](https://github.com/openshift/assisted-service/blob/2688136ed6398480e15eb6575466c265f92983e2/swagger.yaml#L4688), the ignored host validations and ignored cluster validations should be a parameter in that object

Reply:
Sorry, I disagree, we made a purposeful decision to exclude this from the cluster api

Reply:
the suggestion from @avishayt was not to include it in the cluster object because it's not a supported api

Reply:
Alright. Next time please mention such non-obvious decisions in the commit message

Reply:
That's a fair point, I'll amend the commit message now.